tests/test_consistency_checker.py

```diff
@@ -104,6 +105,15 @@ def test_evt_bad_container_name():
     )
 
 
+def test_duplicate_execs_in_container():
+    container = Container("foo", execs={Exec(("ls", "-l"), return_code=0), Exec(("ls", "-l"), return_code=1)})
+    assert_inconsistent(
```

Comment:
I'd rather raise a ValueError in this case.
WDYT?

Reply:
Hmm, the Scenario pattern is generally for the consistency checker to handle making sure that the state is logically consistent and consistent with the meta. If it was a `ValueError` where would you raise that, in `__init__`? Or `__post_init__` given that this is a dataclass?